operators/multiclusterobservability/manifests/base/config/metrics_allowlist.yaml

```diff
@@ -167,6 +167,24 @@ data:
         expr: sum(sum(sum(kube_pod_container_resource_requests{resource=\"memory\"}) by (pod,namespace,container) * on(pod,namespace) group_left(phase) max(kube_pod_status_phase{phase=~\"Running|Pending|Unknown\"} >0) by (pod,namespace,phase)) by (pod,namespace,phase))
       - record: sli:apiserver_request_duration_seconds:trend:1m
         expr: sum(increase(apiserver_request_duration_seconds_bucket{job=\"apiserver\",service=\"kubernetes\",le=\"1\",verb=~\"POST|PUT|DELETE|PATCH\"}[1m])) / sum(increase(apiserver_request_duration_seconds_count{job=\"apiserver\",service=\"kubernetes\",verb=~\"POST|PUT|DELETE|PATCH\"}[1m]))
+      - record: container_cpu_cfs_periods_total:5m:sum
+        expr: sum(increase(container_cpu_cfs_periods_total{endpoint=\"https-metrics\", container!=\"POD\"}[5m]))
+      - record: container_cpu_cfs_throttled_periods_total:5m:sum
+        expr: sum(increase(container_cpu_cfs_throttled_periods_total{endpoint=\"https-metrics\", container!=\"POD\"}[5m]))
+      - record: container_memory_cache:sum
+        expr: sum(container_memory_cache{endpoint=\"https-metrics\", container!=\"\"})
+      - record: container_memory_rss:sum
+        expr: sum(container_memory_rss{endpoint=\"https-metrics\", container!=\"\"})
+      - record: container_memory_swap:sum
+        expr: sum(container_memory_swap{endpoint=\"https-metrics\", container!=\"\"})
+      - record: container_memory_working_set_bytes:sum
+        expr: sum(container_memory_working_set_bytes{endpoint=\"https-metrics\", container!=\"\"})
+      - record: kube_pod_container_resource_limits:sum
+        expr: sum(kube_pod_container_resource_limits{endpoint=\"https-main\", container!=\"\"})
+      - record: kube_pod_container_resource_requests:sum
+        expr: sum(kube_pod_container_resource_requests{endpoint=\"https-main\", container!=\"\"})
+      - record: kube_resourcequota:hard:scalar
```

Comment:
kube_resourcequota:hard:scalar
which high level dashboard requires this metrics? 
I can only find the metrics kube_resourcequota in Kubernetes / Compute Resources / Namespace (Pods) and Kubernetes / Compute Resources / Namespace (Workloads), both them are the dashboards for container/pod

Reply:
Correct, I removed `kube_resourcequota:hard:scalar`.